ci/deps/pyspark-min.yml

```diff
@@ -1,5 +0,0 @@
-# Need to import double-conversion below otherwise `import pyarrow` fails with ImportError: libdouble-conversion.so.3
-double-conversion
```

Comment:
How are these dependency handled now?

Reply:
There's no explicit handling of them. Happy to add it back if necessary, but pyspark tests are all passing. Is there a specific configuration that isn't passing? If so, can you describe it? Happy to add more changes to address if needed.

Reply:
FYI, this patch includes an upgrade of pyspark from 2->3, is that problematic? Happy to downgrade if it is.